dot_local/bin/executable_nix-update.tmpl

```diff
@@ -124,10 +124,12 @@ function command_update {
   eval_command "nix flake update $HOME_MANAGER_HOME"
   eval_command "home-manager switch"
 
+  {{- if ne "wsl" .ostype }}
```

Comment:
Bit annoying that this is needed but can't be bothered to figure it out - windows / wsl is proving nearly more annoying than useful...

Reply:
This was added to prevent changes being made to the lock file that would be reverted by crostini - turns out this was due to WSL running a newer version of nix than crostini

This has more details - https://github.com/tapayne88/dotfiles/issues/60